.github/workflows/django.yml

```diff
@@ -0,0 +1,91 @@
+name: Django CI
+
+on: [push, pull_request]
+
+jobs:
+  postgres:
+
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: postgres
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          # Maps tcp port 5432 on service container to the host
+          - 5432:5432
+
+    name: Postgres - Python ${{ matrix.python-version }}, Django ${{ matrix.django-version }} (Allowed Failures - ${{ matrix.django-version == '3.1.0' || matrix.django-version == 'https://github.com/django/django/archive/master.tar.gz' }} )
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+        django-version: ['2.2.0', '3.0.0', '3.1.0', 'https://github.com/django/django/archive/master.tar.gz']
```

Comment:
Any reason not to use the string `master` here? we're having to repeat this URL in several places, instead we could just have it once in the install step.

Reply:
Because I didn't revisit this after deciding to split the installation into two conditional steps, rather than always using the value from the matrix.